tools/clang/tools/dxclib/dxc.cpp

```diff
@@ -644,7 +644,7 @@ int DxcContext::VerifyRootSignature() {
   IFT(pContainerBuilder->AddPart(hlsl::DxilFourCC::DFCC_RootSignature,
                                  pRootSignature));
   CComPtr<IDxcOperationResult> pOperationResult;
-  IFT(pContainerBuilder->SerializeContainer(&pOperationResult));
+  pContainerBuilder->SerializeContainer(&pOperationResult);
```

Comment:
We shouldn't have removed the `IFT` here.  Failing HRESULT indicates a serious error preventing `pOperationResult` construction.  We should not be accessing it in that case.